snap/info.go

```diff
@@ -103,6 +103,9 @@ type PlaceInfo interface {
 
 	// UserExposedHomeDir returns the snap's new home directory under ~/Snap.
 	UserExposedHomeDir(home string) string
+
+	// BinaryGlobs returns globs that matches all snap binaries.
+	BinaryGlobs() []string
```

Comment:
Not sure about the naming here

Reply:
the consistent name with AppInfo.WrapperPath would be WrapperGlobs but then I would expect these to include also the dir

Reply:
Should I leave it at `BinaryGlobs` to avoid confusion about the output of `WrapperGlobs`, no?

Reply:
BinaryNameGlobs might also be a better name then, as the other globs we have so far all contains dirs